README.rst

```diff
@@ -185,11 +185,12 @@ For more information on archiving, see the
 Samples
 -------
 
-There are two sample applications included in this repository. To get going as fast as possible, clone the whole
+There are three sample applications included in this repository. To get going as fast as possible, clone the whole
 repository and follow the Walkthroughs:
 
 - `HelloWorld <sample/HelloWorld/README.md>`_
 - `Archiving <sample/Archiving/README.md>`_
+- `Cloud API <sample/Cloud/README.md>`_
```

Comment:
If it helps your deliberations, my pull request for cloud API docs is just talking about the features by what they are, rather than giving it some feature-y/product-y title: Adding sections "Registering callbacks to receive session events" and "Session event callbacks"